### PR TITLE
More Interesting Polymorphed Animal Types

### PR DESCRIPTION
**Pull request by Moja10**
# Document the changes in your pull request

Shakes up the animals that are valid options to polymorph into. Smoothes the bottom of the barrel by removing chicken, mouse, and butterfly, aka the animals you turn into and instantly die as. Adds some more interesting mobs that are never seen and would fit in right in a wiz round, like eyeballs, clown goats, spaceproof cat, etc. Adds chocobos, and raw prophets as animals that have the ability to help the crew. Adds some more mob types the wizard might have to worry about creating, like Clown Hulks, raw prophets, and healing lightgeists (not chappy spawned). 

Also, since the mutant fat clown was added, I made it less awful to be. Has more health, not disgustingly and worthlessly slow, but injects laughter on attack, not mindbreaker. 

# Wiki Documentation

If this info is on the wiki, please update to reflect. But AFAIK its not. 

# Changelog

:cl:  
tweak: Wizards polymorph crewmembers into a different set of animals now

/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/magic.dm [hunk 1]
```diff
@@ -200,10 +200,14 @@
 		if("animal")
 			var/path = pick(/mob/living/simple_animal/hostile/carp,
 							/mob/living/simple_animal/hostile/bear,
-							/mob/living/simple_animal/hostile/mushroom,
+							/mob/living/simple_animal/hostile/carp/eyeball,
+							/mob/living/simple_animal/hostile/eldritch/raw_prophet,
```
Comment (tattax) [inline code/modules/projectiles/projectile/magic.dm:204]: evil

Comment (Moja10) [reply code/modules/projectiles/projectile/magic.dm:204]: I want more mobs that could end up hurting the wizard in the long run, a salty former crewmember turned 20 view range telepathic spy sounds like a good way to make wizards be a little more careful with polymorph

File: code/modules/projectiles/projectile/magic.dm [hunk 2]
```diff
@@ -218,17 +222,19 @@
...
 							/mob/living/simple_animal/parrot,
 							/mob/living/simple_animal/pet/dog/corgi,
 							/mob/living/simple_animal/crab,
 							/mob/living/simple_animal/pet/dog/pug,
 							/mob/living/simple_animal/pet/cat,
-							/mob/living/simple_animal/mouse,
-							/mob/living/simple_animal/chicken,
+							/mob/living/simple_animal/pet/cat/space,
+							/mob/living/simple_animal/chocobo,
+							/mob/living/simple_animal/hostile/retaliate/goose/vomit,
```
Comment (Moltijoe) [inline code/modules/projectiles/projectile/magic.dm]: sorry, no birdboat, this one is buggy currently

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/magic.dm (hunk 2 above)
Comment (Moja10) [reply code/modules/projectiles/projectile/magic.dm]: Removed birdboat

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**